RediSearchClient/Converters/BoolToStringConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace RediSearchClient.Converters
+{
+    /// <summary>
+    /// RediSearch does not natively support boolean types, therefore we need to convert booleans to strings in this case
+    /// </summary>
+    internal sealed class BoolToStringConverter : JsonConverter<bool>
```

Comment:
I actually haven't tried out indexing booleans as tags yet, might resolve this issue and we dont need that converter

Reply:
Removed as Tags are better suited to handle booleans